serverless.yml

```diff
@@ -119,6 +121,12 @@ resources:
         EvaluationPeriods: 1
         AlarmActions:
           - Ref: snsTopic
+    logQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: aqts-capture-etl-rdb-log-${self:provider.stage}
+        MessageRetentionPeriod: 1209600
+        VisibilityTimeout : 180
```

Comment:
What's the purpose of this log queue?

Reply:
I'm similarly confused - do we want to use an SNS or SES topic instead?  As written, I think this would populate an SQS queue with log information, but nothing would be done with it.